### DISCUSSION TITLE
Update the version of JQuery

### DISCUSSION THREAD

**Discussion by Tigerlady** [MasaCMS/MasaCMS#17] [Ideas]
This is a suggestion to update the version of jquery in Masa because it throws a security issue with Google Core Vitals.  I have updated all of my older mura sites but if we are to use Masa in the future this is a key item.

**Comment by guustnieuwenhuis** [comment]
What implementation of jQuery are you talking about? The one in the theme, admin interface or both?

**Comment by Tigerlady** [comment]
Actually both of them.  Since it was discontinued for updates I have gone ahead and made updates on the sites we manage in both places but thought it was something that should be looked into. 

Thank you.
